lib/page-object.rb

```diff
@@ -144,7 +146,7 @@ def navigate_to(url)
   # Returns the text of the current page
   #
   def text
-    platform.text
```

Comment:
because it can be rooted at an element, the text of a page is now the text of that root element (which defaults to the <html> tag if none is provided)